app/helpers/UpscanMessageHelper.scala

```diff
@@ -41,4 +45,12 @@ object UpscanMessageHelper {
   def applyMessage(msgKey: String)(implicit messages: Messages): String = {
     messages(msgKey)
   }
+
+  def getPluralOrSingular(total: Int, arg: Int)(msgForSingular: String, msgForPlural: String)(implicit messages: Messages, user: UserRequest[_]): Html = {
```

Comment:
Is there a need for `user: UserRequest[_]`?

Reply:
Removed it.